### PR TITLE
Intime cosmic CI test

### PR DESCRIPTION
**Pull request by rennney**
Add intime cosmic ci test
Several memory/CPU adjustments

### CODE REVIEW

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

File: test/ci/icarus_ci_intimecosmic_gen_quick_test_icaruscode.fcl [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+source: @erase
+
+# Be as strict as possible with random seeds: GENIE is wild!
+physics.producers.generator.Seed: 63214
+
+# Override GENIE flux file configuration
+physics.producers.generator.FluxCopyMethod: "DIRECT"
+physics.producers.generator.MaxFluxFileMB: 20
+physics.producers.generator.FluxSearchPaths: "/cvmfs/sbn.osgstorage.org/pnfs/fnal.gov/usr/sbn/persistent/stash/physics/beam/GENIE/BNB/standard/v01_00/"
+
```
Comment (PetrilloAtWork) [inline test/ci/icarus_ci_intimecosmic_gen_quick_test_icaruscode.fcl:17]: This looks like GENIE settings, which should not be in `CORSIKAGen` configuration.

**Review by PetrilloAtWork — COMMENTED** [review]
Nothing strikes me as wrong on a first glance.
There are spurious settings in the generation stage, but they should not cause problems.

**Comment by rennney** [comment]
trigger build

**Comment by rennney** [comment]
trigger build

**Pull request merged**